main.css

```diff
@@ -0,0 +1,76 @@
+@import url(./reset.css);
+@import url(./components/components.css);
+
+/* html {
+    font-size: 62.5%;
+} */
+
+/* @media only screen and (max-width: 800px){
+	html {
+        font-size: 50%;
+    }
+} */
+
+body {
+    
+    background-color: white;
+}
+
+header {
+    background-color: var(--primary-color);
+    padding: 0.5rem; 
+    display: flex;
+    justify-content: space-between;    
+}
+
+.logo-img {
+    width: 10rem;
+    height: 4rem;
+    padding: -0.5rem;
+}
+
+
+nav, 
+.socials {
+    display: flex;
+    gap: 1rem;
+    font-size: 1.2rem;
+    align-items: center;
+}
+
+
+.hero-img {
+    width: 100vw;
+    height: auto;
+}
+
+
+main {
+    display: flex;
+    flex-direction: row;
+    font-size: 1.2rem;
+}
+
+aside {
```

Comment:
use classes insted of using tags  its best practice.

Reply:
This is in reset.css file, we use reset.css to modify/remove any default CSS added by the browser.  I want this to be universal for all the mentioned selectors which is why I am not using any class selector.

Reply:
I have used class selectors in main.css which are specific for the block

Reply:
ok